custom_components/waste_collection_schedule/waste_collection_schedule/source/wolverhampton_gov_uk.py

```diff
@@ -0,0 +1,98 @@
+import requests
+from bs4 import BeautifulSoup
+from datetime import datetime
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Wolverhampton City Council"
+DESCRIPTION = "Source for Wolverhampton City Council waste collection."
+URL = "https://www.wolverhampton.gov.uk"
+EXTRA_INFO = [
+    {
+        "title": "Find My Nearest",
+        "description": "Find your waste collection schedule",
+        "url": "https://www.wolverhampton.gov.uk/find-my-nearest",
+    }
+]
```

Comment:
This does not really make sense EXTRA_INFO is used to add pseudo sources that can be found in the README and in the HA GUI configuration but adding the additioanl `Find My Nearest` source (linking to this one) does not seem to make any sense here